website/docs/en/guide/migration/storybook.mdx

```diff
@@ -1,53 +1,52 @@
-# Migrate Storybook
+# Migrating Storybook
 
-If you are using the React framework of [Storybook 7.0](https://storybook.js.org/) and building with Webpack 5, you can replace the `@storybook/react-webpack5` with [@modern-js/storybook](https://www.npmjs.com/package/@modern-js/storybook), which supports Rspack as an out-of-the-box underlying build tool. Documentation can be referred to at [Modern.js - Using Storybook](https://modernjs.dev/en/guides/advanced-features/using-storybook.html).
+If you are using React / Vue with [Storybook 8.0](https://storybook.js.org/) and building with webpack 5, you can replace the `@storybook/react-webpack5` build with [storybook-rsbuild](https://github.com/rspack-contrib/storybook-rsbuild), which is implemented based on Rsbuild and uses Rspack as the underlying build tool. It supports out-of-the-box use, and the documentation can be found at [storybook-rsbuild](https://github.com/rspack-contrib/storybook-rsbuild).
```

Comment:
No need to emphasize the version here

```suggestion
If you are using React / Vue with [Storybook](https://storybook.js.org/) and building with webpack 5, you can replace the `@storybook/react-webpack5` build with [storybook-rsbuild](https://github.com/rspack-contrib/storybook-rsbuild), which is implemented based on Rsbuild and uses Rspack as the underlying bundler. It supports out-of-the-box use, and the documentation can be found at [storybook-rsbuild](https://github.com/rspack-contrib/storybook-rsbuild).
```

Reply:
I'd prefer to document sb^8 here, the storybook-builder-rsbuild is build on top of 8.0 and not guaranteed to work with sb7. Since sb8 hasn't been released for long and lots of user are still on sb7, it's better to document it out.

<img width="694" alt="image" src="https://github.com/web-infra-dev/rspack/assets/12322740/026ca626-2722-4932-89b7-8aece665314c">